scholia/app/templates/venue.html

```diff
@@ -261,6 +261,39 @@
 ORDER BY DESC(?year)
 LIMIT 500
  `
+ retractionsJournalSparql =  `
+SELECT DISTINCT
+  ?retracted_work ?retracted_workLabel
+  ?date
+  ?citing_work ?citing_workLabel
+WITH {
+  # Find retracted papers indicated by instance of retracted paper, 
+  # by retraction notice property or by significant event
+  SELECT DISTINCT ?retracted_work WHERE {
+    { ?retracted_work wdt:P31 wd:{{ q }} }
+    UNION
+    { ?retracted_work wdt:P5824 [] }
+    UNION
+    { ?retracted_work wdt:P793 wd:Q45203135 }
+  }
+} AS %retracted_works
+WHERE {
+  INCLUDE %retracted_works
+  ?citing_work wdt:P2860 ?retracted_work ; wdt:P1433 wd:Q180445 .
```

Comment:
wd:Q180445 should be wd:{{ q }}

Reply:
Done
(I'll push the change together with the other changes)